assets/src/block-templates/deep-dive-topic/template.js

```diff
@@ -0,0 +1,34 @@
+const {__} = wp.i18n;
+
+const template = () => ([
+  ['core/group', {}, [
+    ['planet4-block-templates/page-header', {
+      titlePlaceholder: __('Page header title', 'planet4-blocks'),
+      mediaPosition: 'right',
+    }],
+    ['core/spacer', {height: '64px'}],
+    ['planet4-block-templates/side-image-with-text-and-cta', {
+      title: __('The problem', 'planet4-blocks'),
+    }],
+    ['planet4-block-templates/side-image-with-text-and-cta', {
+      title: __('What can be done', 'planet4-blocks'),
+      mediaPosition: 'right',
+    }],
+    ['planet4-blocks/covers', {
+      title: __('How you can help', 'planet4-blocks'),
+      cover_type: 'take-action',
+    }],
+    ['planet4-blocks/articles', {
```

Comment:
We should keep in mind that the Articles and Covers blocks haven't been moved to the theme, so right now if the plugin is disabled these do not work in the template... However I see that we have already moved several other patterns to the theme that use the Covers block, so it's probably not an issue for now.

Reply:
Good catch, thanks!